in_toto/runlib.py

```diff
@@ -665,9 +666,29 @@ def in_toto_run(name, material_list, product_list, link_cmd_args,
       materials=materials_dict, products=products_dict, command=link_cmd_args,
       byproducts=byproducts, environment=environment)
 
+  LOG.info("Creating statement...")
```

Comment:
IMO we don't touch runlib at all in this PR. Let this one just add the Statement layer. A future PR can then add support for link predicates (https://github.com/in-toto/attestation/blob/main/spec/predicates/link.md) and add a flag to runlib to generate links using the Statement layer rather than the current format. We've also got to be mindful of backwards compatibility with runlib and verifylib and this change is certainly a breaking change.